web/impact/impact/settings.py

```diff
@@ -270,6 +270,9 @@ class Base(Configuration):
 
     CMS_FILE_ROOT = '/var/www/cms-files'
 
+    IMAGE_RESIZE_HOST = "https://dl4fx6jt7wkin.cloudfront.net"
```

Comment:
should probably be an env var

Reply:
And I'd consider adding a Route 53 DNS record (within the cloudformation stack?) and give this link a readable name that makes sense (e.g. has staging/prod in it).

Reply:
That's a good idea.

Reply:
After some effort, created a ticket for this.  https://masschallenge.atlassian.net/browse/AC-5794